.vitepress/config.ts

```diff
@@ -65,4 +68,17 @@ export default defineConfig({
     //     'https://github.com/iExecBlockchainComputing/documentation-tools/edit/feature/migrate-to-vitepress/:path',
     // },
   },
+  vite: {
+    plugins: [
+      AutoImport({
+        include: [/\.vue$/, /\.md$/],
+        resolvers: [ElementPlusResolver({ ssr: true })],
+      }),
+      Components({
+        dirs: ['components'],
+        include: [/\.vue$/, /\.md$/],
+        resolvers: [ElementPlusResolver({ ssr: true })],
+      }),
```

Comment:
All components in `/components` folder will be available without explicitly importing them.
Example with `RequiredYesBadge.vue`